ingestion/src/metadata/data_quality/interface/sqlalchemy/sqa_test_suite_interface.py

```diff
@@ -49,23 +49,25 @@ def __init__(
         self,
         service_connection_config: DatabaseConnection,
         ometa_client: OpenMetadata,
+        sampler: SamplerInterface,
         table_entity: Table = None,
-        sqa_metadata=None,
+        orm_table=None,
     ):
-        super().__init__()
-        self.ometa_client = ometa_client
-        self.table_entity = table_entity
-        self.service_connection_config = service_connection_config
+        super().__init__(
+            service_connection_config,
+            ometa_client,
+            sampler,
+            table_entity,
+        )
         self.create_session()
-        self._table = self._convert_table_to_orm_object(sqa_metadata)
```

Comment:
the ORM table is now an input argument https://github.com/open-metadata/OpenMetadata/pull/18610/files#diff-e8a8a48b444e6246572ad762618b52eed69b4b7a75ffba896cf8314f56c15990R132

Before we sent the `sqa_metadata` object and let the interfaces (profiler/test suite) create the ORM. Now, since the sampler needs the ORM table too, we create it at the `source` and pass it as an arg to the 3 interfaces